source/parser.hera

```diff
@@ -834,7 +834,9 @@ ObjectBindingPattern
 ObjectBindingPatternContent
   # NOTE: Added indentation based binding properties
   NestedBindingProperties
-  BindingProperty*:props BindingRestProperty?:rest BindingProperty*:after ->
+  BindingPropertyList*:props BindingRestProperty?:rest BindingPropertyList*:after ->
+    props = props.flat()
+    after = after.flat()
```

Comment:
It seems like `BindingPropertyList` already has a `+` so each `BindingPropertyList*` could be replaced by `BindingPropertyList?`, right? And then you could omit the `.flat()`.

Reply:
This needs the `*` to absorb multiple lines of `Nested`.

Reply:
Maybe I'm missing something, but it seems like (at least in this case) there's no `Nested` being called, it's just a rule with `+` which can't actually consume `EOS`.